File: clients/rust/src/anonymous_schema14.rs
// AnonymousSchema14 represents a AnonymousSchema14 model.
#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct AnonymousSchema14 {
    #[serde(rename="additionalProperties", skip_serializing_if = "Option::is_none")]
    additional_properties: Option<std::collections::HashMap<String, serde_json::Value>>,
}

impl AnonymousSchema14 {
    pub fn new(additional_properties: Option<std::collections::HashMap<String, serde_json::Value>>) -> AnonymousSchema14 {
        AnonymousSchema14 {
        additional_properties,
        }
    }
}

File: clients/typescript/src/AnonymousSchema_14.ts

export class AnonymousSchema_14 {
  private _additionalProperties?: Map<string, any>;

  constructor(input: {
    additionalProperties?: Map<string, any>,
  }) {
    this._additionalProperties = input.additionalProperties;
  }

  get additionalProperties(): Map<string, any> | undefined { return this._additionalProperties; }
  set additionalProperties(additionalProperties: Map<string, any> | undefined) { this._additionalProperties = additionalProperties; }
}
